### ISSUE TITLE
New issue template

### ISSUE THREAD

**Issue by vmarchaud** [Unitech/pm2#2745]
Based on https://github.com/npm/npm/blob/latest/.github/issue_template.md
```
#### I'm opening this issue because:

  - [ ] pm2 isn't behaving like it should do.
  - [ ] pm2 is doing something I don't understand.
  - [ ] pm2 is crashing.
  - [ ] Other (_see at the end for feature requests_):

#### What's going wrong?

<!--
    Please a explain what you are expecting and whats actually happening
-->

#### How can we reproduce the problem?

<!--
    Please a complete description of how to reproduce the problem.
    Include a gist of your npm-debug.log file.
    If you've never used gist.github.com, start here:
      https://github.com/EmmaRamirez/how-to-submit-your-npm-debug-log
-->

### supporting information:

 - `pm2 -v` prints:
 - `node -v` prints:
 - (UNIX) `uname -a` output or (Windows) version:

<!--
    Thank you for contributing to PM2 !

    - Please check we don't already resolved your problem:
      https://github.com/Unitech/pm2/search?type=Issues

    - Also ensure that we don't document the problem in our documentation:
      http://pm2.keymetrics.io/docs/usage/quick-start/

    For feature requests, delete the above and uncomment the section following this one. But first, review the existing feature requests
    and make sure there isn't one that already describes the feature
    you'd like to see added:
      https://github.com/Unitech/pm2/issues?q=is%3Aopen+is%3Aissue+label%3A%22T%3A+Feature%22
-->

<!--

#### What's the feature?

#### What problem is the feature intended to solve?

#### Is the absence of this feature blocking you or your team? If so, how?

#### Is this feature similar to an existing feature in another tool?

#### Is this a feature you're prepared to implement, with support from the PM2 maintainers ?

-->
```

**Comment by soyuka** [comment]
👍

**Comment by Unitech** [comment]
LGTM we can change the current template in the #development branch

**Comment by vmarchaud** [comment]
Published under pm2 `2.4.3` !
